### PR TITLE
*: update keyword auto_increment to CAPS

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? 

AUTO_INCREMENT: update keyword auto_increment to CAPS

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs/pull/1750

### Which version does your change affect? 

ALL

### CODE REVIEW

**Comment by yikeke** [comment]
Please help review the changes in DM-related files. @csuzhangxc

**Comment by yikeke** [comment]
Please help review the changes in DM-related files, especially for `auto_increment_id`. @csuzhangxc 
Rest LGTM

File: v2.1/how-to/get-started/data-migration.md [hunk 1]
```diff
@@ -257,7 +257,7 @@ name: dmtask1
 task-mode: all
 is-sharding: true
 enable-heartbeat: true
-ignore-checking-items: ["auto_increment_ID"]
+ignore-checking-items: ["AUTO_INCREMENT_ID"]
```
Comment (csuzhangxc) [inline v2.1/how-to/get-started/data-migration.md]: ditto

File: v2.1/how-to/get-started/data-migration.md [hunk 2]
```diff
@@ -300,7 +300,7 @@ loaders:
 
 * `is-sharding: true`：多个 DM-worker 实例进行同一个任务，这些实例将上游的若干分片合并到一个下游的表中。
 
-* `ignore-checking-items: ["auto_increment_ID"]`：关闭 DM 对上游实例中潜在的自增 ID 冲突的检测。DM 能检测出上游表结构相同、并包含自增列的分片间潜在的列值冲突。通过配置 `auto-increment-increment` 和 `auto-increment-offset` 可使每个 MySQL Server 的 ID 都不重叠，从而避免不同表之间冲突的产生。因此，可以让 DM 关闭对自增 ID 冲突的检测。
+* `ignore-checking-items: ["AUTO_INCREMENT_ID"]`：关闭 DM 对上游实例中潜在的自增 ID 冲突的检测。DM 能检测出上游表结构相同、并包含自增列的分片间潜在的列值冲突。通过配置 `auto-increment-increment` 和 `auto-increment-offset` 可使每个 MySQL Server 的 ID 都不重叠，从而避免不同表之间冲突的产生。因此，可以让 DM 关闭对自增 ID 冲突的检测。
```
Comment (csuzhangxc) [inline v2.1/how-to/get-started/data-migration.md]: ditto

File: v2.1/reference/tools/data-migration/usage-scenarios/shard-merge.md [hunk 3]
```diff
@@ -126,7 +126,7 @@ category: reference
 - 要满足同步需求 #8，首先参考[自增主键冲突处理](/v2.1/reference/tools/data-migration/usage-scenarios/best-practice-dm-shard.md#自增主键冲突处理)来解决冲突，保证在同步到下游时不会因为分表中有相同的主键值而使同步出现异常；然后需要配置 `ignore-checking-items` 来跳过自增主键冲突的检查：
 
     ```yaml
-    ignore-checking-items: ["auto_increment_ID"]
+    ignore-checking-items: ["AUTO_INCREMENT_ID"]
```
Comment (csuzhangxc) [inline v2.1/reference/tools/data-migration/usage-scenarios/shard-merge.md]: ditto

File: v2.1/reference/tools/data-migration/usage-scenarios/shard-merge.md [hunk 4]
```diff
@@ -138,7 +138,7 @@ name: "shard_merge"
 task-mode: all
 meta-schema: "dm_meta"
 remove-meta: false
-ignore-checking-items: ["auto_increment_ID"]
+ignore-checking-items: ["AUTO_INCREMENT_ID"]
```
Comment (csuzhangxc) [inline v2.1/reference/tools/data-migration/usage-scenarios/shard-merge.md]: ditto

File: v3.0/how-to/get-started/data-migration.md [hunk 5]
```diff
@@ -303,7 +303,7 @@ name: dmtask1
 task-mode: all
 is-sharding: true
 enable-heartbeat: true
-ignore-checking-items: ["auto_increment_ID"]
+ignore-checking-items: ["AUTO_INCREMENT_ID"]
```
Comment (csuzhangxc) [inline v3.0/how-to/get-started/data-migration.md]: ditto

File: v3.0/how-to/get-started/data-migration.md [hunk 6]
```diff
@@ -346,7 +346,7 @@ loaders:
 
 * `is-sharding: true`：多个 DM-worker 实例进行同一个任务，这些实例将上游的若干分片合并到一个下游的表中。
 
-* `ignore-checking-items: ["auto_increment_ID"]`：关闭 DM 对上游实例中潜在的自增 ID 冲突的检测。DM 能检测出上游表结构相同、并包含自增列的分片间潜在的列值冲突。通过配置 `auto-increment-increment` 和 `auto-increment-offset` 可使每个 MySQL Server 的 ID 都不重叠，从而避免不同表之间冲突的产生。因此，可以让 DM 关闭对自增 ID 冲突的检测。
+* `ignore-checking-items: ["AUTO_INCREMENT_ID"]`：关闭 DM 对上游实例中潜在的自增 ID 冲突的检测。DM 能检测出上游表结构相同、并包含自增列的分片间潜在的列值冲突。通过配置 `auto-increment-increment` 和 `auto-increment-offset` 可使每个 MySQL Server 的 ID 都不重叠，从而避免不同表之间冲突的产生。因此，可以让 DM 关闭对自增 ID 冲突的检测。
```
Comment (csuzhangxc) [inline v3.0/how-to/get-started/data-migration.md]: ditto

File: v3.0/reference/tools/data-migration/usage-scenarios/shard-merge.md [hunk 7]
```diff
@@ -139,7 +139,7 @@ aliases: ['/docs-cn/tools/dm/shard-merge-scenario/']
     {{< copyable "" >}}
 
     ```yaml
-    ignore-checking-items: ["auto_increment_ID"]
+    ignore-checking-items: ["AUTO_INCREMENT_ID"]
```
Comment (csuzhangxc) [inline v3.0/reference/tools/data-migration/usage-scenarios/shard-merge.md]: ditto

File: v3.0/reference/tools/data-migration/usage-scenarios/shard-merge.md [hunk 8]
```diff
@@ -153,7 +153,7 @@ name: "shard_merge"
 task-mode: all
 meta-schema: "dm_meta"
 remove-meta: false
-ignore-checking-items: ["auto_increment_ID"]
+ignore-checking-items: ["AUTO_INCREMENT_ID"]
```
Comment (csuzhangxc) [inline v3.0/reference/tools/data-migration/usage-scenarios/shard-merge.md]: ditto

File: v3.1/how-to/get-started/data-migration.md [hunk 9]
```diff
@@ -303,7 +303,7 @@ name: dmtask1
 task-mode: all
 is-sharding: true
 enable-heartbeat: true
-ignore-checking-items: ["auto_increment_ID"]
+ignore-checking-items: ["AUTO_INCREMENT_ID"]
```
Comment (csuzhangxc) [inline v3.1/how-to/get-started/data-migration.md]: ditto

File: v3.1/how-to/get-started/data-migration.md [hunk 10]
```diff
@@ -346,7 +346,7 @@ loaders:
 
 * `is-sharding: true`：多个 DM-worker 实例进行同一个任务，这些实例将上游的若干分片合并到一个下游的表中。
 
-* `ignore-checking-items: ["auto_increment_ID"]`：关闭 DM 对上游实例中潜在的自增 ID 冲突的检测。DM 能检测出上游表结构相同、并包含自增列的分片间潜在的列值冲突。通过配置 `auto-increment-increment` 和 `auto-increment-offset` 可使每个 MySQL Server 的 ID 都不重叠，从而避免不同表之间冲突的产生。因此，可以让 DM 关闭对自增 ID 冲突的检测。
+* `ignore-checking-items: ["AUTO_INCREMENT_ID"]`：关闭 DM 对上游实例中潜在的自增 ID 冲突的检测。DM 能检测出上游表结构相同、并包含自增列的分片间潜在的列值冲突。通过配置 `auto-increment-increment` 和 `auto-increment-offset` 可使每个 MySQL Server 的 ID 都不重叠，从而避免不同表之间冲突的产生。因此，可以让 DM 关闭对自增 ID 冲突的检测。
```
Comment (csuzhangxc) [inline v3.1/how-to/get-started/data-migration.md]: ditto

File: v3.1/reference/tools/data-migration/usage-scenarios/shard-merge.md [hunk 11]
```diff
@@ -138,7 +138,7 @@ category: reference
     {{< copyable "" >}}
 
     ```yaml
-    ignore-checking-items: ["auto_increment_ID"]
+    ignore-checking-items: ["AUTO_INCREMENT_ID"]
```
Comment (csuzhangxc) [inline v3.1/reference/tools/data-migration/usage-scenarios/shard-merge.md]: ditto

File: v3.1/reference/tools/data-migration/usage-scenarios/shard-merge.md [hunk 12]
```diff
@@ -152,7 +152,7 @@ name: "shard_merge"
 task-mode: all
 meta-schema: "dm_meta"
 remove-meta: false
-ignore-checking-items: ["auto_increment_ID"]
+ignore-checking-items: ["AUTO_INCREMENT_ID"]
```
Comment (csuzhangxc) [inline v3.1/reference/tools/data-migration/usage-scenarios/shard-merge.md]: ditto

File: dev/releases/21rc2.md [hunk 13]
```diff
@@ -69,7 +69,7 @@ category: Releases
     - 解决计算 `DateTime` 类型数据的默认值时没有考虑时区的问题 [#7655](https://github.com/pingcap/tidb/pull/7655)
 - DML
     - `InsertOnDuplicateUpdate` 语句设置正确的 `last_insert_id` [#7534](https://github.com/pingcap/tidb/pull/7534)
-    - 减少需要更新 `auto_increment_id` 计数器的情况 [#7515](https://github.com/pingcap/tidb/pull/7515)
+    - 减少需要更新 `AUTO_INCREMENT_id` 计数器的情况 [#7515](https://github.com/pingcap/tidb/pull/7515)
```
Comment (yikeke) [inline dev/releases/21rc2.md]:
```suggestion
    - 减少需要更新 `auto_increment_id` 计数器的情况 [#7515](https://github.com/pingcap/tidb/pull/7515)
```

File: v2.1-legacy/releases/21rc2.md [hunk 14]
```diff
@@ -69,7 +69,7 @@ category: Releases
     - 解决计算 `DateTime` 类型数据的默认值时没有考虑时区的问题 [#7655](https://github.com/pingcap/tidb/pull/7655)
 - DML
     - `InsertOnDuplicateUpdate` 语句设置正确的 `last_insert_id` [#7534](https://github.com/pingcap/tidb/pull/7534)
-    - 减少需要更新 `auto_increment_id` 计数器的情况 [#7515](https://github.com/pingcap/tidb/pull/7515)
+    - 减少需要更新 `AUTO_INCREMENT_id` 计数器的情况 [#7515](https://github.com/pingcap/tidb/pull/7515)
```
Comment (yikeke) [inline v2.1-legacy/releases/21rc2.md]:
```suggestion
    - 减少需要更新 `auto_increment_id` 计数器的情况 [#7515](https://github.com/pingcap/tidb/pull/7515)
```

File: v2.1/releases/21rc2.md [hunk 15]
```diff
@@ -69,7 +69,7 @@ category: Releases
     - 解决计算 `DateTime` 类型数据的默认值时没有考虑时区的问题 [#7655](https://github.com/pingcap/tidb/pull/7655)
 - DML
     - `InsertOnDuplicateUpdate` 语句设置正确的 `last_insert_id` [#7534](https://github.com/pingcap/tidb/pull/7534)
-    - 减少需要更新 `auto_increment_id` 计数器的情况 [#7515](https://github.com/pingcap/tidb/pull/7515)
+    - 减少需要更新 `AUTO_INCREMENT_id` 计数器的情况 [#7515](https://github.com/pingcap/tidb/pull/7515)
```
Comment (yikeke) [inline v2.1/releases/21rc2.md]:
```suggestion
    - 减少需要更新 `auto_increment_id` 计数器的情况 [#7515](https://github.com/pingcap/tidb/pull/7515)
```

File: v3.0/releases/21rc2.md [hunk 16]
```diff
@@ -70,7 +70,7 @@ aliases: ['/docs-cn/releases/21rc2/']
     - 解决计算 `DateTime` 类型数据的默认值时没有考虑时区的问题 [#7655](https://github.com/pingcap/tidb/pull/7655)
 - DML
     - `InsertOnDuplicateUpdate` 语句设置正确的 `last_insert_id` [#7534](https://github.com/pingcap/tidb/pull/7534)
-    - 减少需要更新 `auto_increment_id` 计数器的情况 [#7515](https://github.com/pingcap/tidb/pull/7515)
+    - 减少需要更新 `AUTO_INCREMENT_id` 计数器的情况 [#7515](https://github.com/pingcap/tidb/pull/7515)
```
Comment (yikeke) [inline v3.0/releases/21rc2.md]:
```suggestion
    - 减少需要更新 `auto_increment_id` 计数器的情况 [#7515](https://github.com/pingcap/tidb/pull/7515)
```

File: v3.1/releases/21rc2.md [hunk 17]
```diff
@@ -69,7 +69,7 @@ category: Releases
     - 解决计算 `DateTime` 类型数据的默认值时没有考虑时区的问题 [#7655](https://github.com/pingcap/tidb/pull/7655)
 - DML
     - `InsertOnDuplicateUpdate` 语句设置正确的 `last_insert_id` [#7534](https://github.com/pingcap/tidb/pull/7534)
-    - 减少需要更新 `auto_increment_id` 计数器的情况 [#7515](https://github.com/pingcap/tidb/pull/7515)
+    - 减少需要更新 `AUTO_INCREMENT_id` 计数器的情况 [#7515](https://github.com/pingcap/tidb/pull/7515)
```
Comment (yikeke) [inline v3.1/releases/21rc2.md]:
```suggestion
    - 减少需要更新 `auto_increment_id` 计数器的情况 [#7515](https://github.com/pingcap/tidb/pull/7515)
```

**Pull request merged**